test/unit/vm/disposable_test.dart

```diff
@@ -67,6 +71,23 @@ void main() {
       });
     });
 
+    group('manageDisposer', () {
+      test(
+          'should call callback and accept null return value'
+          'when parent is disposed', () async {
+        thing.testManageDisposer(expectAsync(() => null, count: 1) as Disposer);
```

Comment:
I like to be explicit with that, particularly when the count is the key reason for the expectation.